article/conf/routes

```diff
@@ -40,4 +40,7 @@ GET     /*path/email.emailtxt       controllers.ArticleController.renderEmail(pa
 #  e.g. /theguardian/2015/nov/03/mainsection
 GET     /$publication<(theguardian|theobserver)>/$year<\d\d\d\d>/$month<\w\w\w>/$day<\d\d>/$tail<.+>                 controllers.PublicationController.publishedOn(publication, year, month, day, tail)
 
+GET     /apps/$edition<(uk|us|eu|au|eur|int)>/*path                      controllers.ArticleController.renderAppsArticle(path, edition: String)
```

Comment:
One concern I have here is, do we have any content with an ID that starts with `europe/`, `international/` etc? I think this routing would break for those articles

Reply:
one mitigation for this would be to follow the pattern some existing routes like `/email/headline.txt` have, so we could have:

- `/technology/2021/oct/27/wifi-advice-techscape/apps/uk`

Reply:
resolved this by moving the `/apps` bit to the end of the URL. I've updated the PR description to reflect this.